components/LocationSelectionBanner/LocationSelectionBanner.tsx

```diff
@@ -40,6 +40,7 @@ const LocationSelectionBanner: FunctionComponent<LocationSelectionBannerProps> =
 
   return (
     <SpotlightBanner className={props.className}>
+      <h3 className="spotlight-banner__heading">Add Location</h3>
```

Comment:
Make this optional, and configurable as a prop, with a defaultProp 'Add location'

Reply:
Now optional and passed as a prop; does not appear on homepage unless its set.